lib/Dialect/AIEVec/IR/AIEVecOps.cpp

```diff
@@ -413,10 +413,17 @@ template <typename T> LogicalResult verifyMulFMAOp(T op) {
 
   // The datatype of accumulator must always be greater width
   if (atype.isa<IntegerType>()) {
+    if (!atype.isa<IntegerType>())
```

Comment:
Do you mean ltype or rtype here?  This condition is always false.

Reply:
Ah yep, my bad!